crates/sui-graphql-rpc/src/data.rs

```diff
@@ -30,6 +32,16 @@ pub(crate) type DieselBackend = <Db as QueryExecutor>::Backend;
 pub(crate) type Query<ST, QS, GB> =
     BoxedSelectStatement<'static, ST, FromClause<QS>, DieselBackend, GB>;
 
+#[derive(Clone)]
+pub(crate) struct RawSqlQuery {
```

Comment:
also, nit: This type is growing large enough that it maybe deserves its own module, under `data`?